pyproject.toml

```diff
@@ -43,13 +43,17 @@ ignore = [
 ]
 
 [tool.mypy]
-python_version = "3.8"
+python_version = "3.9"
```

Comment:
IIRC vllm's minimum version is still 3.8

Reply:
+1. Please use python 3.8.